src/pages/get-premium/index.tsx

```diff
@@ -48,35 +46,56 @@ export default function GetPremiumPage() {
       setIsCheckoutSessionLoading(false);
     }
   }
+
+  function handleSubscribeMonthlyButton() {
+    // e.preventDefault();
```

Comment:
Can we just directly attach `handleClickSubscribe('monthly') to the buttons themselves?